test/index.test.js

```diff
@@ -0,0 +1,44 @@
+jest.mock('child_process');
+const childProcess = require('child_process');
+const findPackageManager = require('../index');
+
+// Mock out `child_process` module's `spawnSync` method
+const spawnSyncMock = jest.spyOn(childProcess, 'spawnSync');
+spawnSyncMock.mockImplementation((packageManagerCommand, commandArguments) => {
+    switch (packageManagerCommand) {
+        case 'npm':
+            return {
+                error: undefined // no error thrown
+            };
+        case 'yarn':
+        default:
+            return {
+                error: `Error is being thrown that ${packageManagerCommand} does not exist`
+            };
+    }
+});
+
+describe('Basic test suite', () => {
```

Comment:
@devenbansod 
What do you think about grouping based on functionality?
Like `describe("Something for isAvailable")` and `describe("something for findAll")`

Reply:
Makes sense. Let me try to change it to something like you suggested. 👍

Reply:
@hemal7735 what do you think about the test-suite now? With this, I guess the sentence makes sense now: `isAvailable API should return true if package manager is available`

```
> findall-pkg-mgr@1.1.0 test /Users/debansod/personal/findall-pkg-mgr
> jest

 PASS  test/index.test.js
  isAvailable API
    ✓ should return true if package manager is available (5ms)
    ✓ should return false if package manager is NOT available (1ms)
  findAll API
    ✓ should return all available package managers (1ms)
```